### DISCUSSION TITLE
Where is the pipeline-demo.yaml file ?

### DISCUSSION THREAD

**Discussion by chengyayu** [megaease/easegress#51] [Q&A] [answered]
I started the easegress server according to the set up guide, and test it, but I didn’t find any pipeline-demo.yaml files. 
Then I created it and used the following command to update the configuration:
```
./bin/egctl object update -f pipline-demo.yaml
```
but an error was reported:
```
Error: 0: Not Found
```

**Comment by benja-wu** [comment, marked as answer]
The example YAML files are in `https://github.com/megaease/easegress/tree/main/example/config` this dir.  :-)

**Reply by chengyayu** [reply]
Thank you for your reply. I build a file with wrong name.
